projects/pgai/tests/vectorizer/cli/test_compatibility.py

```diff
@@ -45,3 +45,30 @@ def test_080_vectorizer_definition(
         with conn.cursor() as cur:
             cur.execute("SELECT * FROM blog_embedding_store;")
             assert len(cur.fetchall()) == 4
+
+
+@pytest.mark.postgres_params(ai_extension_version="0.8.0")
+def test_errors_table_compatibility(
```

Comment:
Copied from https://github.com/timescale/pgai/pull/734/ to quickly validate the fix atm